matrix-meetings-bot/src/IEventContentParams.ts

```diff
@@ -20,6 +20,7 @@ import { v4 as uuiv4 } from 'uuid';
 export interface IEventContentParams {
   room_id?: string;
   base32_room_id?: string;
+  base32_room_id50?: string;
```

Comment:
We should also update the documentation: https://github.com/nordeck/matrix-meetings/blob/main/docs/configuration.md#default_events_config